compiler/src/iree/compiler/Dialect/Stream/Transforms/ScheduleExecution.cpp

```diff
@@ -162,6 +162,12 @@ struct ExecutePartitionBuilder {
       if (affinityOp.getAffinityAttr() == partition->affinity) {
         affinityOp.setAffinityAttr(nullptr);
       }
+
+      auto streamableOp =
```

Comment:
```suggestion
      // Any op that prefers being cloned to consumers doesn't need to specify
      // an affinity as it can be no more specialized than any valid consumer
      // of it. Clearing the affinity here may allow the target a greater
      // ability to optimize the operation.
      auto streamableOp =
```